browser/ui/sidebar/sidebar_browsertest.cc

```diff
@@ -1145,7 +1145,6 @@ IN_PROC_BROWSER_TEST_F(SidebarBrowserTestWithAIChat, TabSpecificPanel) {
 
   // Delete Tab 0 and check model doesn't have active index.
   tab_model()->DetachAndDeleteWebContentsAt(0);
-  EXPECT_FALSE(!!model()->active_index());
```

Comment:
Hmm, we don't need delete this line.

Reply:
You're right, that was left over from an older change that's no longer needed. Removed this entire commit, thank you!